ibis/expr/types/temporal.py

```diff
@@ -339,6 +339,46 @@ def strftime(self, format_str: str) -> ir.StringValue:
         -------
         StringValue
             Formatted version of `arg`
+
+        Examples
+        --------
+        >>> from datetime import date
+        >>> import ibis
+        >>> ibis.options.interactive = True
+        >>> t = ibis.memtable(
+        ...     [
+        ...         date(2020, 10, 5),
+        ...         date(2020, 11, 10),
+        ...         date(2020, 12, 15),
+        ...     ],
+        ...     schema=ibis.Schema({"date_col": "date"}),
```

Comment:
```suggestion
        ...     {"date_col": [
        ...         date(2020, 10, 5),
        ...         date(2020, 11, 10),
        ...         date(2020, 12, 15),
        ...     ]},
```

You'll need to run `ruff` again to format this correctly, if you accept the suggestion.